webrtc-stats.html

```diff
@@ -1340,7 +1339,8 @@
               </dt>
               <dd>
                 <p>
-                  Represents the RTT computed by the STUN connectivity checks [[!STUN-PATH-CHAR]].
+                  Represents the RTT computed by the STUN connectivity checks
+                  [[!STUN-PATH-CHAR]] as well as consent checks [[!RFC7675]].
```

Comment:
I wanted to confirm that it's the intention. Because this value will be different from the value exposed by the RTP stack (the combination of RTCP SR and RR). Even though the media maybe flowing on that particular candidate pair.